Fixtures/CFamilyTargets/CUBSan/include/main.h

```diff
@@ -0,0 +1 @@
+int main(int argc, char **argv);
```

Comment:
The header is not required.

Reply:
Yeah I agree, if anything this seems like the linking testing is happening in the Swift PR and should be sufficient. Should we just remove the fixture entirely then?